baseplate/lib/events.py

```diff
@@ -32,105 +23,6 @@
 MAX_QUEUE_SIZE = 10000
 
 
-# pylint: disable=pointless-string-statement,no-init
-class FieldKind(Enum):
```

Comment:
I'm not super familiar with this part of baseplate. I'm assuming this is going away because v2 events are a thing now?

Reply:
Yeah, the v1 events are on their way out. I confirmed with data that we're good to delete this from 1.0.